docs/preview/features/message-pumps/customization.md

```diff
@@ -14,6 +14,8 @@ We allow several customizations while implementing your own message pump.
 
 ## Control custom deserialization
```

Comment:
I would move this after "Filter messages based on message context" & "Fallback message handling" because it will be less used, unless you want to keep this for alphabetically sorting (which is ok)